core/src/main/java/com/sequenceiq/distrox/v1/distrox/controller/DistroXV1Controller.java

```diff
@@ -750,4 +755,11 @@ public FlowIdentifier modifyProxyInternal(@TenantAwareParam String crn, String p
     public FlowIdentifier rotateSecrets(@RequestObject DistroXSecretRotationRequest request) {
         return stackOperationService.rotateSecrets(request.getCrn(), request.getSecrets(), request.getExecutionType());
     }
+
+    @Override
+    @CheckPermissionByResourceCrn(action = DATALAKE_HORIZONTAL_SCALING)
+    public FlowIdentifier refreshDatalakeConfig(@ResourceCrn String crn) {
```

Comment:
What's the usecase for this? How would the user know when to invoke this command?

Reply:
we should do this instead of customer, seems like an internal stuff, why don't we include this call as an internal call in the horizontal scale flow?